src/main/scala/group/holmes/analytics/actors/Core.scala

```diff
@@ -0,0 +1,27 @@
+package group.holmes.analytics.actors
+
+import java.io.File
+
+import akka.actor.{ Actor, ActorLogging, ActorRef, Props }
+import com.typesafe.config.Config
+
+
+object Core {
+	def props(cfg: Config, analyticEngineManager: ActorRef, analyticServiceManager: ActorRef): Props = { Props(new Core(cfg, analyticEngineManager, analyticServiceManager)) }
+}
+
+class Core(cfg: Config, analyticEngineManager: ActorRef, analyticServiceManager: ActorRef) extends Actor with ActorLogging {
+	override def preStart(): Unit = log.info("Core started")
+	override def postStop(): Unit = log.info("Core stopped")
+	override def receive = Actor.emptyBehavior
+
+	// create the WebServer
+	val webserver = context.actorOf(WebServer.props(cfg.getConfig("webserver")))
```

Comment:
What about naming this "API"?  I say that as you will eventually host the Holmes' Frontend website in this project.